chainer/function_hooks/cupy_memory_profile.py

```diff
@@ -129,10 +129,12 @@ def summary(self):
     def _choose_unit(self, size):
         """Choose optimal unit."""
         denomi = 1
+        if size <= 0:
+            return denomi, self._units[0]
         for unit in self._units[:-1]:
-            if size / denomi >= 1:
+            if size / (denomi * 1024.0) < 1:
```

Comment:
How about fixing the previous line?
`for unit in self._units:`

Reply:
Sorry, I couldn't get your point. Will that simplify this code?

Reply:
I'm sorry.  I was confused with `self._units[::-1]`.